src/generate/device.rs

```diff
@@ -146,6 +146,13 @@ pub fn render(
     let generic_file = std::str::from_utf8(include_bytes!("generic.rs"))?;
     if generic_mod {
         writeln!(File::create("generic.rs")?, "{}", generic_file)?;
+
+        out.extend(quote! {
+            #[allow(unused_imports)]
+            use generic::*;
+            ///Common register and bit access and modify traits
```

Comment:
Annoyingly plain comments like this don't make it through to the generated code, so to keep it in the output we need to use the `doc` attribute instead.

```suggestion
            #[doc="Common register and bit access and modify traits"]
```